src/commands/client-no-touch.json

```
@@ -0,0 +1,38 @@
+{
+    "NO-TOUCH": {
+        "summary": "Set client will or not touch LFU/LRU stats when performing read operations for the current connection.",
```

```suggestion
        "summary": "Set client will or not touch LFU/LRU stats for the current connection.",
```

LFU/LRU is not touched for write operations either, right?

---

Yeah I missed this...